tests/src/Module/Api/Mastodon/PushSubscriptionTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * @copyright Copyright (C) 2010-2022, the Friendica project
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Test\src\Module\Api\Mastodon;
+
+use Friendica\Test\src\Module\Api\ApiTest;
+
+class PushSubscriptionTest extends ApiTest
+{
+	/**
+	 * Test the api_account_verify_credentials() function.
+	 *
+	 * @return void
+	 */
+	public function testApiAccountVerifyCredentials()
+	{
+		$this->markTestIncomplete('Needs mocking of whole applictaions/Apps first');
+
+		// $this->useHttpMethod(Router::POST);
+		//
+		// $response = (new PushSubscription(DI::app(), DI::l10n(), DI::baseUrl(), DI::args(), DI::logger(), DI::profiler(), DI::apiResponse(), DI::mstdnSubscription(), DI::mstdnError(), []))
+		// 	->run();
+		//
+		// $json = $this->toJson($response);
+		// print_r($json);
+		//
+		// $this->assertEquals(1,1);
+	}
+
+	/**
+	 * Test the api_account_verify_credentials() function without an authenticated user.
+	 *
+	 * @return void
+	 */
+	public function testApiAccountVerifyCredentialsWithoutAuthenticatedUser()
```

Comment:
You already added `: void` in `PushSubscription` class, can you please add it here?